packages/itwin/measure-tools/src/api/Measurement.ts

```diff
@@ -57,14 +59,17 @@ export namespace DrawingMetadata {
       return undefined;
     const origin = obj.origin?.toJSONXY();
     const extents = obj.extents?.toJSONXY();
+    const masterOrigin = obj.sheetToWorldTransform?.masterOrigin.toJSONXYZ();
+    const sheetTov8Drawing = obj.sheetToWorldTransform?.sheetTov8Drawing.toJSON();
+    const v8DrawingToDesign = obj.sheetToWorldTransform?.v8DrawingToDesign.toJSON();
     if (origin !== undefined)
-      return { origin, extents, worldScale: obj.worldScale, drawingId: obj.drawingId };
+      return { origin, extents, worldScale: obj.worldScale, drawingId: obj.drawingId, sheetToWorldTransform: (masterOrigin !== undefined && sheetTov8Drawing !== undefined && v8DrawingToDesign !== undefined) ? {masterOrigin, sheetTov8Drawing, v8DrawingToDesign}: undefined };
     return undefined;
   }
 
   export function fromJSON(json: DrawingMetadataProps): DrawingMetadata {
 
-    return { origin: Point2d.fromJSON(json.origin), worldScale: json.worldScale, drawingId: json.drawingId, extents: Point2d.fromJSON(json.extents)};
+    return { origin: Point2d.fromJSON(json.origin), worldScale: json.worldScale, drawingId: json.drawingId, extents: Point2d.fromJSON(json.extents), sheetToWorldTransform: { masterOrigin: Point3d.fromJSON(json.sheetToWorldTransform?.masterOrigin), sheetTov8Drawing: Transform.fromJSON(json.sheetToWorldTransform?.sheetTov8Drawing), v8DrawingToDesign: Transform.fromJSON(json.sheetToWorldTransform?.v8DrawingToDesign)}};
```

Comment:
Since the `sheetToWorldTransform` in `DrawingMetadata` is optional, I would make sure that `json.sheetToWorldTransform` is defined before definitely assigning it using `fromJSON` methods (which will return an identity transform).